applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -21,6 +21,9 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 ====
```

Comment:
@Preeticp Can you please check that I haven't re-added anything that shouldn't be here while I was fixing my conflicts? Thanks